moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryDataTrackerSignals.js

```diff
@@ -17,21 +23,20 @@ const useProjectSignals = (project) =>
     [project]
   );
 
-const ProjectSummaryDataTrackerSignals = ({ classes, project }) => {
+const ProjectSummaryDataTrackerSignals = ({ project }) => {
   const signals = useProjectSignals(project);
 
   return (
     <>
-      <Grid item xs={12} className={classes.fieldGridItem}>
-        <Typography className={classes.fieldLabel}>Signal IDs</Typography>
+      <Grid item xs={12} sx={fieldGridItem}>
+        <Typography sx={fieldLabel}>Signal IDs</Typography>
         <Box display="flex" justifyContent="flex-start">
           <ProjectSummaryLabel
-            className={classes.fieldLabelDataTrackerLink}
+            sxProp={fieldLabelDataTrackerLink}
+            spanSxProp={fieldLabelTextSpanNoBorder}
             text={
               signals.length > 0 ? <RenderSignalLink signals={signals} /> : "-"
             }
-            classes={classes}
```

Comment:
no need to pass these classes down anymore, only classes that override the classes imported in the component